src/json-schema/bundled.json

```diff
@@ -2271,6 +2271,233 @@
+												"properties": {
+													"Bucket": {
+														"type": "string",
+														"description": "A string that specifies the Amazon S3 bucket name to export the results to.",
+														"minLength": 1
+													},
+													"Prefix": {
+														"type": "string",
+														"description": "A string that specifies the Amazon S3 object prefix to export the results to.",
+														"minLength": 1
+													}
+												},
+												"required": [
+													"Bucket",
+													"Prefix"
+												]
+											}
+										},
+										"required": [
+											"Resource",
+											"Parameters"
+										]
+									},
+									"ItemProcessor": {
+										"type": "object",
+										"description": "A JSON object that defines a state machine which will process each item or batch of items of the array.",
+										"properties": {
+											"ProcessorConfig": {
+												"type": "object",
+												"description": "A JSON object that specifies the Map state processing mode and definition.",
+												"properties": {
+													"Mode": {
+														"type": "string",
+														"description": "The Map state processing mode.",
+														"minLength": 1,
+														"enum": [
+															"DISTRIBUTED",
+															"INLINE"
+														]
+													},
+													"ExecutionType": {
+														"type": "string",
+														"description": "The execution type for the Map workflow.",
+														"minLength": 1,
+														"enum": [
+															"EXPRESS",
+															"STANDARD"
+														]
+													}
+												},
+												"required": [
+													"Mode",
+													"ExecutionType"
+												]
+											}
+										},
+										"required": [
+											"ProcessorConfig"
+										]
+									},
+									"ToleratedFailurePercentage": {
+										"type": "integer",
+										"description": "An integer that provides an upper bound on the percentage of items that may fail.",
+										"minimum": 0,
+										"maximum": 100
+									},
+									"ToleratedFailurePercentagePath": {
+										"$ref": "#/properties/States/additionalProperties/oneOf/0/allOf/1/properties/ResultPath/oneOf/0",
+										"description": "A reference path to an integer that provides an upper bound on the percentage of items that may fail."
+									},
+									"ToleratedFailureCount": {
+										"type": "integer",
+										"description": "An integer that provides an upper bound on how many items may fail.",
+										"minimum": 0
+									},
+									"ToleratedFailureCountPath": {
+										"$ref": "#/properties/States/additionalProperties/oneOf/0/allOf/1/properties/ResultPath/oneOf/0",
+										"description": "A reference path to an integer that provides an upper bound on how many items may fail."
+									},
 									"MaxConcurrency": {
```

Comment:
`Label` is also listed under https://docs.aws.amazon.com/step-functions/latest/dg/concepts-asl-use-map-state-distributed.html. 

Will also need to add to `validationSchema.ts`.

Reply:
Added in another commit.